### PR TITLE
Initial checkin for 10 AWS specific doc only app pages.

### PR DESCRIPTION
**Pull request by npande**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/amazon-aws/amazon-appflow.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Log and Metric types
+* [CloudWatch Metrics](https://docs.aws.amazon.com/appflow/latest/userguide/monitoring-cloudwatch.html)
+* [CloudTrail Logs](https://docs.aws.amazon.com/appflow/latest/userguide/appflow-cloudtrail-logs.html)
+
+
+## Setup
+This section has instructions for collecting logs and metrics for the Sumo Logic's Amazon AppFlow integration.
+
+### Configure metrics collection
+* Collect CloudWatch Metrics with namespace AWS/AppFlow using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/AppFlow metrics and dimensions refer [Amazom AppFlow CloudWatch metrics](https://docs.aws.amazon.com/appflow/latest/userguide/monitoring-cloudwatch.html)
```
Comment (JV0812) [inline docs/integrations/amazon-aws/amazon-appflow.md:21]:
```suggestion
* Collect CloudWatch Metrics with namespace AWS/AppFlow using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/AppFlow metrics and dimensions refer to [Amazom AppFlow CloudWatch metrics](https://docs.aws.amazon.com/appflow/latest/userguide/monitoring-cloudwatch.html).
```

File: docs/integrations/amazon-aws/amazon-appflow.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+<img src={useBaseUrl('img/integrations/amazon-aws/amazon-appflow-logo.png')} alt="Thumbnail icon" width="50"/>
+
+Amazon AppFlow is a fully-managed integration service that enables you to securely exchange data between software as a service (SaaS) applications and AWS services, such as Amazon Simple Storage Service (Amazon S3) and Amazon Redshift. For more details, refer to the [AWS documentation](https://docs.aws.amazon.com/appflow/latest/userguide/what-is-appflow.html).
+
+## Log and Metric types
+* [CloudWatch Metrics](https://docs.aws.amazon.com/appflow/latest/userguide/monitoring-cloudwatch.html)
+* [CloudTrail Logs](https://docs.aws.amazon.com/appflow/latest/userguide/appflow-cloudtrail-logs.html)
+
+
```
Comment (JV0812) [inline docs/integrations/amazon-aws/amazon-appflow.md:16]:
```suggestion
```

File: docs/integrations/amazon-aws/amazon-appstream2.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+## Log and Metric types
+* [CloudWatch Metrics](https://docs.aws.amazon.com/appstream2/latest/developerguide/monitoring.html#monitoring-with-cloudwatch)
+* [CloudTrail Logs](https://docs.aws.amazon.com/appstream2/latest/developerguide/logging-using-cloudtrail.html)
+
+
+## Setup
+This section has instructions for collecting logs and metrics for the Sumo Logic's Amazon AppStream 2.0 integration.
+
+### Configure metrics collection
+* Collect CloudWatch Metrics with namespace AWS/AppStream using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/AppStream metrics and dimensions refer [Amazon AppStream 2.0 CloudWatch metrics](https://docs.aws.amazon.com/appstream2/latest/developerguide/monitoring.html#monitoring-with-cloudwatch)
```
Comment (JV0812) [inline docs/integrations/amazon-aws/amazon-appstream2.md:21]:
```suggestion
* Collect CloudWatch Metrics with namespace AWS/AppStream using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/AppStream metrics and dimensions refer to [Amazon AppStream 2.0 CloudWatch metrics](https://docs.aws.amazon.com/appstream2/latest/developerguide/monitoring.html#monitoring-with-cloudwatch).
```

File: docs/integrations/amazon-aws/amazon-athena.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+<img src={useBaseUrl('img/integrations/amazon-aws/amazon-athena-logo.png')} alt="Thumbnail icon" width="50"/>
+
+Amazon Athena is an interactive query service that makes it easy to analyze data directly in Amazon Simple Storage Service (Amazon S3) using standard SQL. With a few actions in the AWS Management Console, you can point Athena at your data stored in Amazon S3 and begin using standard SQL to run ad-hoc queries and get results in seconds. For more details, refer to the [AWS documentation](https://docs.aws.amazon.com/athena/latest/ug/what-is.html).
+
+## Log and Metric types
+* [CloudWatch Metrics](https://docs.aws.amazon.com/athena/latest/ug/query-metrics-viewing.html#athena-cloudwatch-metrics-table)
+* [CloudTrail Logs](https://docs.aws.amazon.com/athena/latest/ug/monitor-with-cloudtrail.html)
+
+
```
Comment (JV0812) [inline docs/integrations/amazon-aws/amazon-athena.md:16]:
```suggestion
```

File: docs/integrations/amazon-aws/amazon-athena.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+## Setup
+This section has instructions for collecting logs and metrics for the Sumo Logic's Amazon Athena integration.
+
+### Configure metrics collection
+* Collect CloudWatch Metrics with namespace AWS/Athena using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/Athena metrics and dimensions refer [Amazon Athena CloudWatch metrics](https://docs.aws.amazon.com/athena/latest/ug/query-metrics-viewing.html#athena-cloudwatch-metrics-table)
```
Comment (JV0812) [inline docs/integrations/amazon-aws/amazon-athena.md:21]:
```suggestion
* Collect CloudWatch Metrics with namespace AWS/Athena using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/Athena metrics and dimensions refer to [Amazon Athena CloudWatch metrics](https://docs.aws.amazon.com/athena/latest/ug/query-metrics-viewing.html#athena-cloudwatch-metrics-table).
```

File: docs/integrations/amazon-aws/amazon-cloudfront.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+## Log and Metric types
+* [CloudWatch Metrics](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/monitoring-using-cloudwatch.html)
+* [Access Logs](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/AccessLogs.html)
+* [Real-time logs](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/real-time-logs.html)
+* CloudWatch Logs
+  * [Edge Function Logs](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/edge-functions-logs.html)
+* [CloudTrail Logs](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/logging_using_cloudtrail.html)
+
```
Comment (JV0812) [inline docs/integrations/amazon-aws/amazon-cloudfront.md:19]:
```suggestion
```

File: docs/integrations/amazon-aws/amazon-cloudfront.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+## Setup
+This section has instructions for collecting logs and metrics for the Sumo Logic's Amazon CloudFront integration.
+
+### Configure metrics collection
+* Collect CloudWatch Metrics with namespace [AWS/CloudFront](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/monitoring-using-cloudwatch.html) using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/CloudFront metrics and dimensions refer [Amazon CloudFront CloudWatch metrics](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/programming-cloudwatch-metrics.html)
```
Comment (JV0812) [inline docs/integrations/amazon-aws/amazon-cloudfront.md:25]:
```suggestion
* Collect CloudWatch Metrics with namespace [AWS/CloudFront](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/monitoring-using-cloudwatch.html) using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/CloudFront metrics and dimensions refer to [Amazon CloudFront CloudWatch metrics](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/programming-cloudwatch-metrics.html).
```

File: docs/integrations/amazon-aws/aws-apprunner.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+<img src={useBaseUrl('img/integrations/amazon-aws/aws-apprunner-logo.png')} alt="Thumbnail icon" width="50"/>
+
+AWS App Runner is a fully managed container application service that lets you build, deploy, and run containerized web applications and API services without prior infrastructure or container experience. For more details, refer to the [AWS documentation](https://docs.aws.amazon.com/apprunner/latest/dg/what-is-apprunner.html).
+
+## Log and Metric types
+* [CloudWatch Metrics](https://docs.aws.amazon.com/apprunner/latest/dg/monitor-cw.html)
+* [CloudWatch Logs](https://docs.aws.amazon.com/apprunner/latest/dg/monitor-cwl.html)
+* [CloudTrail Logs](https://docs.aws.amazon.com/apprunner/latest/dg/monitor-ct.html)
+
+
```
Comment (JV0812) [inline docs/integrations/amazon-aws/aws-apprunner.md:17]:
```suggestion
```

File: docs/integrations/amazon-aws/aws-apprunner.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+## Setup
+This section has instructions for collecting logs and metrics for the Sumo Logic's AWS App Runner integration.
+
+### Configure metrics collection
+* Collect CloudWatch Metrics with namespace AWS/AppRunner using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/AppRunner metrics and dimensions refer [AWS App Runner CloudWatch metrics](https://docs.aws.amazon.com/apprunner/latest/dg/monitor-cw.html)
```
Comment (JV0812) [inline docs/integrations/amazon-aws/aws-apprunner.md:22]:
```suggestion
* Collect CloudWatch Metrics with namespace AWS/AppRunner using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/AppRunner metrics and dimensions refer to [AWS App Runner CloudWatch metrics](https://docs.aws.amazon.com/apprunner/latest/dg/monitor-cw.html).
```

File: docs/integrations/amazon-aws/aws-appsync.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+<img src={useBaseUrl('img/integrations/amazon-aws/aws-appsync-logo.png')} alt="Thumbnail icon" width="50"/>
+
+AWS AppSync enables developers to connect their applications and services to data and events with secure, serverless and high-performing GraphQL and Pub/Sub APIs. For more details, refer to the [AWS documentation](https://docs.aws.amazon.com/appsync/latest/devguide/what-is-appsync.html).
+
+## Log and Metric types
+* [CloudWatch Metrics](https://docs.aws.amazon.com/appsync/latest/devguide/monitoring.html#cw-metrics)
+* [CloudWatch Logs](https://docs.aws.amazon.com/appsync/latest/devguide/monitoring.html#cwl)
+* [CloudTrail Logs](https://docs.aws.amazon.com/appsync/latest/devguide/cloudtrail-logging.html)
+
```
Comment (JV0812) [inline docs/integrations/amazon-aws/aws-appsync.md:16]:
```suggestion
```

File: docs/integrations/amazon-aws/aws-appsync.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+## Setup
+This section has instructions for collecting logs and metrics for the Sumo Logic's AWS AppSync integration.
+
+### Configure metrics collection
+* Collect CloudWatch Metrics with namespace AWS/AppSync using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/AppSync metrics and dimensions refer [AWS AppSync CloudWatch metrics](https://docs.aws.amazon.com/appsync/latest/devguide/monitoring.html#cw-metrics)
```
Comment (JV0812) [inline docs/integrations/amazon-aws/aws-appsync.md:22]:
```suggestion
* Collect CloudWatch Metrics with namespace AWS/AppSync using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/AppSync metrics and dimensions refer to [AWS AppSync CloudWatch metrics](https://docs.aws.amazon.com/appsync/latest/devguide/monitoring.html#cw-metrics).
```

File: docs/integrations/amazon-aws/aws-backup.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/amazon-aws/aws-backup-logo.png')} alt="Thumbnail icon" width="50"/>
+
+AWS Backup is a fully-managed service that makes it easy to centralize and automate data protection across AWS services, in the cloud, and on premises. Using this service, you can configure backup policies and monitor activity for your AWS resources in one place. It allows you to automate and consolidate backup tasks that were previously performed service-by-service, and removes the need to create custom scripts and manual processes. For more details, refer to the [AWS documentation](https://docs.aws.amazon.com/aws-backup/latest/devguide/whatisbackup.html).
+
+## Log and Metric types
+* [CloudWatch Metrics](https://docs.aws.amazon.com/aws-backup/latest/devguide/cloudwatch.html#monitoring-metrics-with-cloudwatch)
+* [CloudTrail Logs](https://docs.aws.amazon.com/aws-backup/latest/devguide/logging-using-cloudtrail.html)
+
```
Comment (JV0812) [inline docs/integrations/amazon-aws/aws-backup.md:15]:
```suggestion
```

File: docs/integrations/amazon-aws/aws-backup.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+
+## Setup
+This section has instructions for collecting logs and metrics for the Sumo Logic's AWS Backup integration.
+
+### Configure metrics collection
+* Collect CloudWatch Metrics with namespace AWS/Backup using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/Backup metrics and dimensions refer [AWS Backup CloudWatch metrics](https://docs.aws.amazon.com/aws-backup/latest/devguide/cloudwatch.html#monitoring-metrics-with-cloudwatch)
```
Comment (JV0812) [inline docs/integrations/amazon-aws/aws-backup.md:21]:
```suggestion
* Collect CloudWatch Metrics with namespace AWS/Backup using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/Backup metrics and dimensions refer to [AWS Backup CloudWatch metrics](https://docs.aws.amazon.com/aws-backup/latest/devguide/cloudwatch.html#monitoring-metrics-with-cloudwatch).
```

File: docs/integrations/amazon-aws/aws-certificate-manager.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/amazon-aws/aws-certificate-manager-logo.png')} alt="Thumbnail icon" width="50"/>
+
+AWS Certificate Manager (ACM) handles the complexity of creating, storing, and renewing public and private SSL/TLS X.509 certificates and keys that protect your AWS websites and applications. You can provide certificates for your integrated AWS services either by issuing them directly with ACM or by importing third-party certificates into the ACM management system. ACM certificates can secure singular domain names, multiple specific domain names, wildcard domains, or combinations of these. ACM wildcard certificates can protect an unlimited number of subdomains. You can also export ACM certificates signed by AWS Private CA for use anywhere in your internal PKI. For more details, refer to the [AWS documentation](https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html).
+
+## Log and Metric types
+* [CloudWatch Metrics](https://docs.aws.amazon.com/acm/latest/userguide/cloudwatch-metrics.html)
+* [CloudTrail Logs](https://docs.aws.amazon.com/acm/latest/userguide/cloudtrail.html)
+
```
Comment (JV0812) [inline docs/integrations/amazon-aws/aws-certificate-manager.md:15]:
```suggestion
```

File: docs/integrations/amazon-aws/aws-certificate-manager.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+
+## Setup
+This section has instructions for collecting logs and metrics for the Sumo Logic's AWS Certificate Manager integration.
+
+### Configure metrics collection
+* Collect CloudWatch Metrics with namespace AWS/CertificateManager using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/CertificateManager metrics and dimensions refer [AWS Certificate Manager CloudWatch metrics](https://docs.aws.amazon.com/acm/latest/userguide/cloudwatch-metrics.html)
```
Comment (JV0812) [inline docs/integrations/amazon-aws/aws-certificate-manager.md:21]:
```suggestion
* Collect CloudWatch Metrics with namespace AWS/CertificateManager using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/CertificateManager metrics and dimensions refer to [AWS Certificate Manager CloudWatch metrics](https://docs.aws.amazon.com/acm/latest/userguide/cloudwatch-metrics.html).
```

File: docs/integrations/amazon-aws/aws-chatbot.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
+---
+id: aws-chatbot
+title: AWS Chatbot
+description: Learn about the collection process for the AWS Chatbot service.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/amazon-aws/aws-chatbot-logo.png')} alt="Thumbnail icon" width="50"/>
+
+AWS Chatbot is an AWS service that enables DevOps and software development teams to use messaging program chat rooms to monitor and respond to operational events in their AWS Cloud. AWS Chatbot processes AWS service notifications from Amazon Simple Notification Service (Amazon SNS), and forwards them to chat rooms so teams can analyze and act on them immediately, regardless of location.
```
Comment (JV0812) [inline docs/integrations/amazon-aws/aws-chatbot.md:10]:
```suggestion
AWS Chatbot is an AWS service that enables DevOps and software development teams to use messaging program chat rooms to monitor and respond to operational events in their AWS Cloud. AWS Chatbot processes AWS service notifications from Amazon Simple Notification Service (Amazon SNS) and forwards them to chat rooms so teams can analyze and act on them immediately, regardless of location.
```

File: docs/integrations/amazon-aws/aws-chatbot.md [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+* [CloudWatch Metrics](https://docs.aws.amazon.com//chatbot/latest/adminguide/monitoring-cloudwatch.html#available-cloudwatch-metrics)
+* [CloudWatch Logs](https://docs.aws.amazon.com//chatbot/latest/adminguide/cloudwatch-logs.html)
+* [CloudTrail Logs](https://docs.aws.amazon.com//chatbot/latest/adminguide/logging-using-cloudtrail.html)
+
+
+## Setup
+This section has instructions for collecting logs and metrics for the Sumo Logic's AWS Chatbot integration.
+
+### Configure metrics collection
+* Collect CloudWatch Metrics with namespace AWS/Chatbot using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/Chatbot metrics and dimensions refer [AWS Chatbot CloudWatch metrics](https://docs.aws.amazon.com//chatbot/latest/adminguide/monitoring-cloudwatch.html#available-cloudwatch-metrics)
```
Comment (JV0812) [inline docs/integrations/amazon-aws/aws-chatbot.md:26]:
```suggestion
* Collect CloudWatch Metrics with namespace AWS/Chatbot using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/Chatbot metrics and dimensions refer to [AWS Chatbot CloudWatch metrics](https://docs.aws.amazon.com//chatbot/latest/adminguide/monitoring-cloudwatch.html#available-cloudwatch-metrics).
```

File: docs/integrations/amazon-aws/aws-private-certificate-authority.md [hunk 18]
```diff
@@ -0,0 +1,33 @@
+---
+id: aws-private-certificate-authority
+title: AWS Private Certificate Authority
+description: Learn about the collection process for the AWS Private Certificate Authority service.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/amazon-aws/aws-private-certificate-authority-logo.png')} alt="Thumbnail icon" width="50"/>
+
+AWS Private CA enables creation of private certificate authority (CA) hierarchies, including root and subordinate CAs, without the investment and maintenance costs of operating an on-premises CA. Your private CAs can issue end-entity X.509 certificates useful in scenarios including:
```
Comment (JV0812) [inline docs/integrations/amazon-aws/aws-private-certificate-authority.md:10]:
```suggestion
AWS Private CA enables the creation of private certificate authority (CA) hierarchies, including root and subordinate CAs, without the investment and maintenance costs of operating an on-premises CA. Your private CAs can issue end-entity X.509 certificates useful in scenarios including:
```

File: docs/integrations/amazon-aws/aws-private-certificate-authority.md [hunk 19]
```diff
@@ -0,0 +1,33 @@
...
+## Log and Metric types
+* [CloudWatch Metrics](https://docs.aws.amazon.com/privateca/latest/userguide/PcaCloudWatch.html)
+* [CloudTrail Logs](https://docs.aws.amazon.com/privateca/latest/userguide/PcaCtIntro.html)
+
+
+## Setup
+This section has instructions for collecting logs and metrics for the Sumo Logic's AWS Private Certificate Authority integration.
+
+### Configure metrics collection
+* Collect CloudWatch Metrics with namespace AWS/ACMPrivateCA using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/ACMPrivateCA metrics and dimensions refer [AWS Private Certificate Authority CloudWatch metrics](https://docs.aws.amazon.com/privateca/latest/userguide/PcaCloudWatch.html)
```
Comment (JV0812) [inline docs/integrations/amazon-aws/aws-private-certificate-authority.md:29]:
```suggestion
* Collect CloudWatch Metrics with namespace AWS/ACMPrivateCA using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/ACMPrivateCA metrics and dimensions refer to [AWS Private Certificate Authority CloudWatch metrics](https://docs.aws.amazon.com/privateca/latest/userguide/PcaCloudWatch.html).
```

**Review by JV0812 — APPROVED** [review]
LGTM

File: docs/integrations/amazon-aws/amazon-appflow.md [hunk 20]
```diff
@@ -0,0 +1,24 @@
...
+### Configure metrics collection
+* Collect **CloudWatch Metrics** with namespace `AWS/AppFlow` using the [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For `AWS/AppFlow` metrics and dimensions, refer to [Amazom AppFlow CloudWatch metrics](https://docs.aws.amazon.com/appflow/latest/userguide/monitoring-cloudwatch.html).
+
+### Configure logs collection
+* Collect [AWS CloudTrail Logs](https://docs.aws.amazon.com/appflow/latest/userguide/appflow-cloudtrail-logs.html) using the [AWS CloudTrail](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source/) source. Amazon AppFlow is integrated with AWS CloudTrail, a service that provides a record of actions taken by a user, role, or AWS service in Amazon AppFlow. CloudTrail captures all API calls for Amazon AppFlow as events, which includes calls from the Amazon AppFlow console and code calls to the Amazon AppFlow API operations. If you create a trail, you can enable continuous delivery of CloudTrail events to an Amazon S3 bucket, including events for Amazon AppFlow.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/amazon-appflow.md:24]:
```suggestion
* Collect [AWS CloudTrail Logs](https://docs.aws.amazon.com/appflow/latest/userguide/appflow-cloudtrail-logs.html) using the [AWS CloudTrail](/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source/) source. Amazon AppFlow is integrated with AWS CloudTrail, a service that provides a record of actions taken by a user, role, or AWS service in Amazon AppFlow. CloudTrail captures all API calls for Amazon AppFlow as events, which includes calls from the Amazon AppFlow console and code calls to the Amazon AppFlow API operations. If you create a trail, you can enable continuous delivery of CloudTrail events to an Amazon S3 bucket, including events for Amazon AppFlow. 
```

Comment (kimsauce) [inline docs/integrations/amazon-aws/amazon-appflow.md:24]:
```suggestion
* Collect [AWS CloudTrail Logs](https://docs.aws.amazon.com/appflow/latest/userguide/appflow-cloudtrail-logs.html) using the [AWS CloudTrail](/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source/) source. Amazon AppFlow is integrated with AWS CloudTrail, a service that provides a record of actions taken by a user, role, or AWS service in Amazon AppFlow. CloudTrail captures all API calls for Amazon AppFlow as events, which includes calls from the Amazon AppFlow console and code calls to the Amazon AppFlow API operations. If you create a trail, you can enable continuous delivery of CloudTrail events to an Amazon S3 bucket, including events for Amazon AppFlow. 
```

File: docs/integrations/amazon-aws/amazon-appstream2.md [hunk 21]
```diff
@@ -0,0 +1,24 @@
...
+## Log and Metric types
+* [CloudWatch Metrics](https://docs.aws.amazon.com/appstream2/latest/developerguide/monitoring.html#monitoring-with-cloudwatch)
+* [CloudTrail Logs](https://docs.aws.amazon.com/appstream2/latest/developerguide/logging-using-cloudtrail.html)
+
+## Setup
+
+You can collect the logs and metrics for Sumo Logic's Amazon AppStream 2.0 integration by following the below steps.
+
+### Configure metrics collection
+* Collect **CloudWatch Metrics** with namespace `AWS/AppStream` using the [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For `AWS/AppStream` metrics and dimensions, refer to [Amazon AppStream 2.0 CloudWatch metrics](https://docs.aws.amazon.com/appstream2/latest/developerguide/monitoring.html#monitoring-with-cloudwatch).
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/amazon-appstream2.md:21]:
```suggestion
* Collect **CloudWatch Metrics** with namespace `AWS/AppStream` using the [AWS Kinesis Firehose for Metrics](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For `AWS/AppStream` metrics and dimensions, refer to [Amazon AppStream 2.0 CloudWatch metrics](https://docs.aws.amazon.com/appstream2/latest/developerguide/monitoring.html#monitoring-with-cloudwatch).
```

File: docs/integrations/amazon-aws/amazon-appstream2.md [hunk 22]
```diff
@@ -0,0 +1,24 @@
...
+
+### Configure logs collection
+* Collect [AWS CloudTrail Logs](https://docs.aws.amazon.com/appstream2/latest/developerguide/logging-using-cloudtrail.html) using the [AWS CloudTrail](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source/) source. Amazon AppStream 2.0 is integrated with AWS CloudTrail, a service that provides a record of actions taken by a user, role, or AWS service in AppStream 2.0. CloudTrail captures API calls for AppStream 2.0 as events, which include calls from the AppStream 2.0 console and code calls to the AppStream 2.0 API operations. If you create a trail, you can enable continuous delivery of CloudTrail events to an Amazon S3 bucket, including events for AppStream 2.0.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/amazon-appstream2.md:24]:
```suggestion
* Collect [AWS CloudTrail Logs](https://docs.aws.amazon.com/appstream2/latest/developerguide/logging-using-cloudtrail.html) using the [AWS CloudTrail](/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source/) source. Amazon AppStream 2.0 is integrated with AWS CloudTrail, a service that provides a record of actions taken by a user, role, or AWS service in AppStream 2.0. CloudTrail captures API calls for AppStream 2.0 as events, which include calls from the AppStream 2.0 console and code calls to the AppStream 2.0 API operations. If you create a trail, you can enable continuous delivery of CloudTrail events to an Amazon S3 bucket, including events for AppStream 2.0. 
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
@npande , there are several instances where you'll need to convert docs links to relative links. I've commented on a few but there are more.

**Comment by npande** [comment]
> @npande , there are several instances where you'll need to convert docs links to relative links. I've commented on a few but there are more.

We are having README.md in the apps repo , we cannot use relative paths. We are using the content of same README.md file in the docs repo. Can docs team update relative links in the documentation repo?

**Comment by JV0812** [comment]
I will take care of this action. While I update the index page I will update these links too.

**Pull request merged**